styles.css

```diff
@@ -0,0 +1,705 @@
+  background-color: var(--primary-light-dark);
+  transition: all 0.5s linear;
+}
+
+.show--header--nav--container {
+  height: 100%;
+}
+
+.header--nav ul {
+  padding: 50px 0;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.header--nav ul a {
+  color: #fff;
+  font-size: 15px;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+}
+
+.nav--close--icon {
+  width: 25px;
+  position: absolute;
+  top: 1%;
+  left: 90%;
+  z-index: 1;
+  cursor: pointer;
+}
+
+.header--nav ul li {
+  padding: 20px;
+}
+
+.header--nav ul a:hover {
+  color: var(--primary-orange);
+}
+
+.header--nav ul li:hover a {
+  color: #000;
+}
+
+.header--nav ul li:hover {
+  background-color: var(--primary-orange);
+}
+
+.header--hero--section {
+  padding: 6rem 20px 4.44rem 20px;
+}
+
+.about--header--section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  padding: 10px;
+}
+
+.hero--greeting {
+  color: rgba(255, 79, 56, 1);
+  font-size: 1.2rem;
+}
+
+.about--hero--greeting {
+  color: rgba(255, 79, 56, 1);
+  font-size: 2rem;
+}
+
+.header--hero--section h1,
+.about--header--section h1 {
+  color: rgba(255, 79, 56, 1);
+  font-size: 2rem;
+  text-transform: uppercase;
+}
+
+.about--header--section h1 {
+  font-size: 2rem;
+}
+
+.hero--description {
+  color: #515151;
+  font-size: 1rem;
+  font-family: Lato, sans-serif;
+  margin-top: 3rem;
+  line-height: 1.7;
+  padding: 0.2rem 1rem;
+  border: 8.497px solid #fff;
+  background: #f8f8f8;
+}
+
+.hero--date {
+  color: #595959;
+  font-size: 1.25rem;
+  font-weight: 900;
+  padding-top: 2.93rem;
+}
+
+.hero--venue {
+  color: #868686;
+  font-size: 0.8125rem;
+  font-weight: 700;
+  padding: 0.7rem 0;
+}
+
+.main--program {
+  background-image: url(images/Prog-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  padding: 40px 20px;
+}
+
+.main--program--header--container,
+.featured--speaker--header--container,
+.main--about--header--container,
+.partners--header--container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.6rem;
+}
+
+.main--about--info {
+  padding: 20px;
+}
+
+.main--about--header--text {
+  color: #838383;
+  text-align: center;
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.3125rem;
+  padding: 15px 0;
+}
+
+.main--program--header,
+.main--about--header {
+  color: #fff;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  position: relative;
+}
+
+.main--about--header {
+  color: #2c2c2c;
+  text-align: center;
+  font-size: 1.25rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.75rem;
+}
+
+.featured--speaker--header {
+  color: #000;
+  font-size: 1.25rem;
+}
+
+.main--program--header--line,
+.featured--speaker--header--line,
+.main--about--header--line,
+.partners--header--line {
+  width: 1.875rem;
+  height: 0.125rem;
+  height: 3px;
+  background: #a04038;
```

Comment:
- [ ] kindly use the correct color here... `#EC5242`
remember the 3 main provided Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`